python/ray/train/examples/horovod/horovod_example.py

```diff
@@ -140,9 +140,6 @@ def train_func(config):
     log_interval = config.get("log_interval", 10)
     use_cuda = config.get("use_cuda", False)
 
-    if use_cuda:
```

Comment:
was this accidentally removed?

Reply:
My understanding is we have this in `setup()` already - so it's duplicate code.

Reply:
Ah yes you're right!